src/main.ts

```diff
@@ -74,7 +74,15 @@ function _getAndValidateArgs(): IIssuesProcessorOptions {
     exemptAllAssignees: core.getInput('exempt-all-assignees') === 'true',
     exemptAllIssueAssignees: _toOptionalBoolean('exempt-all-issue-assignees'),
     exemptAllPrAssignees: _toOptionalBoolean('exempt-all-pr-assignees'),
-    enableStatistics: core.getInput('enable-statistics') === 'true'
+    enableStatistics: core.getInput('enable-statistics') === 'true',
+    ignoreAllActivitiesBeforeStale:
```

Comment:
I believe the latest version of `core` has a `getBooleanInput` which might make this a little nicer to read

Reply:
Based on this code https://github.com/actions/toolkit/blob/8df94d9879bfbc0d59bbf3d26779ecf286108ad7/packages/core/src/core.ts#L113
I think it will throw an error instead of returning undefined and to avoid the throw we need a none-required input and provide a default value which would be... a boolean once again.

You mentioned "minimize the options" ⇾ to use this method we need to get rid of the "common options" providing a configuration for issues AND PRs.
I have a side which say: offer simplicity to the consumer and if you have struggles, take the time to do better code
And the one which say: fuck it, it would be way easier for the code and the options to simply distinguish issue and PRs.

So, in the two cases, I am not happy, but the second choice seems more reasonable, especially because there is and could be, options dedicated to only issues or only PRs and splitting the doc in half between issues and PRs would be neat.

I am willing to provide a big-bang PR to do that, but I want your approval on this before to avoid working on "perhaps".

Finally, for the fact that the feature requests for the new options are growing I think it's totally OK; it means people simply need this, and it's the purpose of this action.
The only drawback is that the issue processor was initially coded as a single file and I try to split it with the new features but still, the workflow is not trivial to understand with the current state IMO.
Having an open-mind to propose a way to split it would be nice!

Reply:
yep that makes sense @C0ZEN. I agree it's good to keep providing value, I just think balancing new features vs maintainability is good to think through.